PR: https://github.com/odoo/odoo/pull/

From: e7d409c98c99941e2aa27f21a452a947c65588ca
From: Laurent Smet
Date: 2016-09-23 11:54:20

Structural Changes: 6
Total Changes: 20

[IMP] base_gengo: Move gengo parameters from company settings to general settings

================================= pseudo patch: =================================

--- a/addons/base_gengo/models/__init__.py
+++ b/addons/base_gengo/models/__init__.py
@@ -2,4 +2,5 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 import ir_translation
-import res_company
\ No newline at end of file
+import res_company
+import res_config
\ No newline at end of file

--- a/None
+++ b/addons/base_gengo/models/res_config.py
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+
+class BaseConfigSettings(models.TransientModel):
+    _inherit = 'base.config.settings'
+
+    gengo_private_key = fields.Text(string="Gengo Private Key", related="company_id.gengo_private_key")
+    gengo_public_key = fields.Text(string="Gengo Public Key", related="company_id.gengo_public_key")
+    gengo_comment = fields.Text(string="Comments", related="company_id.gengo_comment",
+      help="This comment will be automatically be enclosed in each an every request sent to Gengo")
+    gengo_auto_approve = fields.Boolean(string="Auto Approve Translation ?", related="company_id.gengo_auto_approve",
+      help="Jobs are Automatically Approved by Gengo.")
+    gengo_sandbox = fields.Boolean(string="Sandbox Mode",
+      help="Check this box if you're using the sandbox mode of Gengo, mainly used for testing purpose.")
